public/js/main.js

```diff
@@ -11,9 +11,23 @@ function toggleNightMode() {
 	localStorage.setItem("night", (night == "true") ? "false" : "true");
 }
 
-// Switches between themes when a new one is selected
-function switchThemes(){
-	themeName = document.getElementById("theme-selector").value
+function changeTheme(opt) {
+	theme = opt.value;
+	localStorage.setItem("theme", theme);
+	document.getElementById("theme").href = "/css/" + theme;
+	console.log(theme);
+}
+
+*/
+
+
+function switchThemes(themeName=null){
```

Comment:
I'm not sure about what happens when a browser that doesn't support default arguments encounters them. Is is a graceful failure or does it throw an error? I don't have access to any computers with IE, can you test this?